docs/content/docs/features/append-only-table.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "Append Only Table"
+weight: 2
+type: docs
+aliases:
+- /features/append-only-table.html
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Append Only Table
```

Comment:
Could you add the configuration explanation for COW and MOR for append only table?

Reply:
I disagree. Concepts is a category for general ideas, not for detailed documents like "how shall we create a table" or "how to tweak the performance of compaction". We might need a general "table types" page in concepts to introduce what is a primary-key table and what is a append-only table, but absolutely not for detailed examples.

Reply:
@tsreaper, I previously comment that refers to [Table & Query Types](https://hudi.apache.org/docs/table_types/) to introduce the append only table and primary key table. @JingsongLi, WDYT?